content/docs/setup/kubernetes/additional-setup/cni/index.md

```diff
@@ -215,7 +170,7 @@ corresponding application pod annotation key.
 
 ### Logging
 
-The Istio CNI plugin is run by the container runtime process space, and, therefore, the log entries are added within
+The Istio CNI plugin is run by the container runtime process space. Therefore, the log entries are logged by the
 the `kubelet` process.
 
 ## Compatibility with other CNI plugins
```

Comment:
In this section, @sdake had a comment that the `warning` wording was ambiguous.  Could you change "this should have no effect" to "The Istio CNI plugin should have no effect" in this PR.

Reply:
Done.